LICENSE

```diff
@@ -1,7 +1,7 @@
 The MIT License (MIT)
 
 CakePHP(tm) : The Rapid Development PHP Framework (https://cakephp.org)
-Copyright (c) 2005-2018, Cake Software Foundation, Inc. (https://cakefoundation.org)
+Copyright (c) 2005-2022, Cake Software Foundation, Inc. (https://cakefoundation.org)
```

Comment:
```suggestion
Copyright (c) 2005-present, Cake Software Foundation, Inc. (https://cakefoundation.org)
```

Reply:
This should just be the year. The 2005-present thing isn't really a valid copyright.

Reply:
Done